server/monitor/src/main/java/org/apache/accumulo/monitor/Monitor.java

```diff
@@ -282,11 +279,25 @@ public void fetchData() {
           if (client != null) {
             mmi = client.getManagerStats(TraceUtil.traceInfo(), context.rpcCreds());
             retry = false;
+            // Now that Manager is up, set the coordinator host
+            Set<ServerId> managers = context.instanceOperations().getServers(ServerId.Type.MANAGER);
+            if (managers == null || managers.isEmpty()) {
+              throw new IllegalStateException(
+                  "io.getServers returned nothing for Manager, but it's up.");
+            }
+            ServerId manager = managers.iterator().next();
+            Optional<HostAndPort> nextCoordinatorHost =
+                Optional.of(HostAndPort.fromString(manager.toHostPortString()));
+            if (coordinatorHost.isEmpty() || (!nextCoordinatorHost.isEmpty()
+                && !coordinatorHost.orElseThrow().equals(nextCoordinatorHost.orElseThrow()))) {
+              coordinatorHost = nextCoordinatorHost;
```

Comment:
```suggestion
            HostAndPort nextCoordinatorHost = HostAndPort.fromString(manager.toHostPortString());
            if (coordinatorHost.isEmpty()
                || !coordinatorHost.orElseThrow().equals(nextCoordinatorHost)) {
              coordinatorHost = Optional.of(nextCoordinatorHost);
```
Simplified things a bit. `!nextCoordinatorHost.isEmpty()` would always be true I think so I removed that.

Reply:
Thanks,  good catch.